ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -32,12 +32,12 @@ import {
 import { updateVisibleTiers } from 'testing/test-utils';
 
 import {
-  allModules,
+  allInitialModules,
```

Comment:
renamed because this is no longer universal: these original modules are now the "initial registration mode" modules